aspnetcore/signalr/redis-backplane.md

```diff
@@ -54,7 +54,7 @@ This article explains SignalR-specific aspects of setting up a [Redis](https://r
 
 ::: moniker range="> aspnetcore-2.1"
 
-* In the SignalR app, install the `Microsoft.AspNetCore.SignalR.StackExchangeRedis` NuGet package.
+* In the SignalR app, install the `Microsoft.AspNetCore.SignalR.StackExchangeRedis` NuGet package. (This package is for ASP.NET Core 2.2 and later; there is also a `Microsoft.AspNetCore.SignalR.Redis` package, which is available for ASP.NET Core 2.1 and 2.2 but will not be supported in 3.0.)
```

Comment:
Maybe something more like:
"This package depends on StackExchange.Redis 2.X.X. There is also the Microsoft.AspNetCore.SignalR.Redis package which depends on StackExchange.Redis 1.X.X, this package will not be shipping in 3.0. Choose the one appropriate for your applications dependencies."